3rdparty/lua-5.1.3/src/loslib.c

```diff
@@ -40,11 +40,13 @@ static int os_pushresult (lua_State *L, int i, const char *filename) {
 
 
 static int os_execute (lua_State *L) {
+#ifndef ANDROID
```

Comment:
This is because I couldn't fix an issue where `posix_spawn` was not found. I thought about fixing it but I think we don't execute any os commands from lua so I added this for now

Reply:
Yes, iirc `os` is removed from lua that runs on the clients. There's a note about it in https://github.com/mindsnacks/wonder_content/wiki/Localization-considerations (search for `LUA_OSLIBNAME`).